tox.ini

```diff
@@ -0,0 +1,30 @@
+[tox]
+envlist = flake8,
+          py27,
+          py35,
+          py36,
+          py37,
+          py38,
+          pypy,
+          pypy3,
+          coverage
+
+[testenv]
+usedevelop = true
+commands =
+    zope-testrunner --test-path=src []
+extras = test
+
+[testenv:coverage]
+basepython = python3.7
+commands =
+    coverage run -m zope.testrunner --test-path=src []
+    coverage report --fail-under=86
+deps =
+    coverage
+
+[testenv:flake8]
+basepython = python3.6
```

Comment:
It does look a bit weird to have different base pythons for coverage and flake8.

Personally I'd use `basepython = python3`, whichever is the current Python 3 version on the developer's machine.